drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/hmis_csv_cleanup/delete_empty_enrollments.rb

```diff
@@ -0,0 +1,56 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module HmisCsvTwentyTwenty::HmisCsvCleanup
+  class DeleteEmptyEnrollments < Base
+    def cleanup!
+      enrollment_batch = []
+
+      es_with_no_service = enrollment_scope.
+        merge(HmisCsvTwentyTwenty::Importer::Project.es).
+        merge(HmisCsvTwentyTwenty::Importer::Project.night_by_night).
```

Comment:
This could be `merge(HmisCsvTwentyTwenty::Importer::Project.es.night_by_night).`